the_debuginator.h

```diff
@@ -2377,6 +2377,30 @@ void debuginator_move_to_parent(TheDebuginator* debuginator) {
 	}
 }
 
+void debuginator_move_to_root(TheDebuginator* debuginator) {
+	DebuginatorItem* hot_item = debuginator->hot_item;
+	while (hot_item && hot_item != debuginator->root) {
```

Comment:
Well, it turns out I'm an idiot, and forgot collapsing folders isn't a thing yet so I was designing around a things that doesn't exist. I've made it less stupid now, but can't build/test from home.